project/home/templates/home/home_page.html

```diff
@@ -19,9 +19,15 @@ <h1>{{ block.value }}</h1>
         </section>
     {% endif %}
 {% endfor %}
-{% with invite_friends_page=invite_friends_page.specific %}
-    <div class="float-end">
+
+<div class="float-end">
+    {% with invite_friends_page=invite_friends_page.specific %}
         <h2><a href="{% pageurl invite_friends_page %}">{{ invite_friends_page.title }}</a></h2>
-    </div>
-{% endwith %}
+    {% endwith %}
+    <br>
+    {% with chapters_index_page=chapters_index_page.specific %}
```

Comment:
From what I understand, it shouldn't be necessary to use the `.specific` attribute, but I may be mistaken. Let's try without `.specific` and see if this still works.
```suggestion
    {% with chapters_index_page=chapters_index_page %}
```

Reply:
It still works! 😸